code/modules/response_team/ert.dm

```diff
@@ -0,0 +1,369 @@
+// ERTs
+
+#define ERT_TYPE_AMBER		1
+#define ERT_TYPE_RED		2
+#define ERT_TYPE_GAMMA		3
+
+/datum/game_mode
+	var/list/datum/mind/ert = list()
+
+var/list/response_team_members = list()
```

Comment:
This section doesn't seem tabbed into the datum declaration. Am I missing something?

Reply:
Gotcha. Hrm. I don't know enough about it myself.

Worth looking into though.